apps/src/blockly/addons/cdoRendererThrasos.js

```diff
@@ -1,9 +1,9 @@
 import GoogleBlockly from 'blockly/core';
 import CdoPathObject from './cdoPathObjectThrasos';
 import CdoConstantsProvider from './cdoConstantsProvider';
-import {addInlineRowSeparators} from '@blockly/renderer-inline-row-separators';
 
-export class CdoRendererThrasosBase extends GoogleBlockly.thrasos.Renderer {
+export default class CdoRendererThrasosBase extends GoogleBlockly.thrasos
+  .Renderer {
```

Comment:
Is this on a newline intentionally? Or is it just rendering weird in Github?

Reply:
I think it's just due to Prettier determining that the line is now a little too long. Pretty sure that all I did was delete `default` and press save.

Confirmed by trying to manually revert:
![image](https://github.com/code-dot-org/code-dot-org/assets/43474485/4a56ad5a-dbc1-47e1-921a-13a6fb234897)